com.github.dictation_toolbox.Caster.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: com.github.dictation_toolbox.Caster
```

Comment:
you didn't fix it. The filename should have been change. the app-id in that case MUST start with `io.github` as per the documentation.